### PR TITLE
dist/codespell: update the list of ignored words and fix all typos

### PR DESCRIPTION
**Pull request by aabadie**
### Contribution description

This PR updates the codespell list of ignored words so all (current) false positives are removed. It also fixes all other real typos raised by the tool.

### Testing procedure

Run `./dist/tools/codespell/check.sh`. With this PR, you get an empty output (no typo found)

<details><summary>master</summary>

```
$ ./dist/tools/codespell/check.sh
There are typos in the following files:

CODING_CONVENTIONS.md:61: seperate ==> separate
LOSTANDFOUND.md:63: noone ==> no one
LOSTANDFOUND.md:63: noone ==> no one
boards/common/stm32/include/f0/cfg_clock_default.h:34: HSI ==> HIS
boards/common/stm32/include/f0/cfg_clock_default.h:63: HSI ==> HIS
boards/common/stm32/include/f0/cfg_clock_default.h:88: HSI ==> HIS
boards/common/stm32/include/f0/cfg_clock_default.h:115: HSI ==> HIS
boards/common/stm32/include/f1f3/cfg_clock_default.h:39: HSI ==> HIS
boards/common/stm32/include/f1f3/cfg_clock_default.h:68: HSI ==> HIS
boards/common/stm32/include/f1f3/cfg_clock_default.h:94: HSI ==> HIS
boards/common/stm32/include/f1f3/cfg_clock_default.h:125: HSI ==> HIS
boards/common/stm32/include/g0/cfg_clock_default.h:17: HSI ==> HIS
boards/common/stm32/include/g0/cfg_clock_default.h:37: HSI ==> HIS
boards/common/stm32/include/g0/cfg_clock_default.h:66: HSI ==> HIS
boards/common/stm32/include/g0/cfg_clock_default.h:104: HSI ==> HIS
boards/common/stm32/include/g4/cfg_clock_default.h:60: HSI ==> HIS
boards/common/stm32/include/l0l1/cfg_clock_default.h:32: HSI ==> HIS
boards/common/stm32/include/l0l1/cfg_clock_default.h:75: HSI ==> HIS
boards/common/stm32/include/l0l1/cfg_clock_default.h:104: HSI ==> HIS
boards/common/stm32/include/l0l1/cfg_clock_default.h:137: HSI ==> HIS
boards/common/stm32/include/l4/cfg_clock_default.h:76: HSI ==> HIS
boards/derfmega128/doc.txt:13: produkt ==> product
boards/derfmega128/doc.txt:16: produkt ==> product
boards/derfmega128/doc.txt:19: produkt ==> product
boards/derfmega128/doc.txt:22: produkt ==> product
boards/derfmega128/doc.txt:25: produkt ==> product
boards/derfmega128/doc.txt:28: produkt ==> product
boards/derfmega256/doc.txt:11: produkt ==> product
boards/derfmega256/doc.txt:12: produkt ==> product
boards/derfmega256/doc.txt:12: produkt ==> product
boards/dwm1001/doc.txt:12: tranceiver ==> transceiver
boards/dwm1001/include/board.h:77: tranceiver ==> transceiver
boards/dwm1001/include/periph_conf.h:53: tranceiver ==> transceiver
boards/limifrog-v1/doc.txt:37: Inferface ==> Interface
boards/mcb2388/board_init.c:2: Technik ==> Technique
boards/mcb2388/include/adc_params.h:2: Technik ==> Technique
boards/mcb2388/include/board.h:2: Technik ==> Technique
boards/mcb2388/include/gpio_params.h:2: Technik ==> Technique
boards/mcb2388/include/periph_conf.h:2: Technik ==> Technique
boards/nrf52dk/doc.txt:9: messurment ==> measurement
core/include/clist.h:18: circularly ==> circular
core/include/msg_bus.h:58: busses ==> buses
core/include/msg_bus.h:61: busses ==> buses
core/include/msg_bus.h:83: busses ==> buses
core/include/msg_bus.h:131: seach ==> search
core/lifo.c:47: tThe ==> the
cpu/arm7_common/include/arm7_common.h:72: BORD ==> BOARD, BORED, BORDER
cpu/cc26xx_cc13xx/include/cpu_conf_cc26xx_cc13xx.h:167: exmaple ==> example
cpu/cortexm_common/include/cpu_conf_common.h:82: pre-empt ==> preempt
cpu/cortexm_common/include/cpu_conf_common.h:122: efficency ==> efficiency
cpu/cortexm_common/include/cpu_conf_common.h:125: pre-empt ==> preempt
cpu/cortexm_common/vectors_cortexm.c:387: occured ==> occurred
cpu/esp_common/include/esp_common.h:54: implementd ==> implemented
cpu/fe310/include/cpucycle.h:31: hart ==> heart, harm
cpu/fe310/periph/i2c.c:175: transfert ==> transfer, transferred
cpu/fe310/periph/i2c.c:178: transfert ==> transfer, transferred
cpu/kinetis/include/cpu_conf_kinetis.h:108: PTD ==> PDF
cpu/kinetis/periph/ics.c:106: PRES ==> PRESS
cpu/kinetis/periph/rtc.c:40: inout ==> input, in out
cpu/lpc1768/periph/uart.c:117: THRE ==> THREE, THERE, THEIR, THE
cpu/lpc23xx/periph/adc.c:2: Technik ==> Technique
cpu/lpc23xx/periph/dac.c:2: Technik ==> Technique
cpu/lpc23xx/periph/i2c.c:2: Technik ==> Technique
cpu/lpc23xx/periph/pm.c:2: Technik ==> Technique
cpu/native/include/async_read.h:45: funtion ==> function
cpu/native/syscalls.c:463: creat ==> create
cpu/nrf52/radio/nrf802154/nrf802154_radio.c:592: substract ==> subtract
cpu/sam0_common/periph/dac.c:2: Technik ==> Technique
cpu/sam0_common/periph/gpio.c:43: acces ==> access
cpu/sam0_common/periph/timer.c:148: withing ==> within
cpu/stm32/dist/clk_conf/clk_conf.c:186: HSI ==> HIS
cpu/stm32/dist/clk_conf/clk_conf.c:250: hsi ==> his
cpu/stm32/dist/clk_conf/clk_conf.c:251: HSI ==> HIS
cpu/stm32/dist/clk_conf/clk_conf.c:303: HSI ==> HIS
cpu/stm32/dist/clk_conf/clk_conf.h:171: hsi ==> his
cpu/stm32/dist/clk_conf/clk_conf.h:171: HSI ==> HIS
cpu/stm32/dist/clk_conf/clk_conf.h:181: HSI ==> HIS
cpu/stm32/dist/clk_conf/clk_conf.h:316: hsi ==> his
cpu/stm32/dist/clk_conf/clk_conf.h:344: hsi ==> his
cpu/stm32/dist/clk_conf/clk_conf.h:372: hsi ==> his
cpu/stm32/dist/clk_conf/clk_conf.h:400: hsi ==> his
cpu/stm32/dist/clk_conf/clk_conf.h:413: hsi ==> his
cpu/stm32/dist/clk_conf/clk_conf.h:441: hsi ==> his
cpu/stm32/dist/clk_conf/clk_conf.h:455: hsi ==> his
cpu/stm32/dist/clk_conf/clk_conf.h:469: hsi ==> his
cpu/stm32/dist/clk_conf/clk_conf.h:483: hsi ==> his
cpu/stm32/dist/clk_conf/clk_conf.h:497: hsi ==> his
cpu/stm32/dist/clk_conf/clk_conf.h:512: hsi ==> his
cpu/stm32/dist/clk_conf/clk_conf.h:526: hsi ==> his
cpu/stm32/dist/clk_conf/clk_conf.h:541: hsi ==> his
cpu/stm32/dist/clk_conf/clk_conf.h:556: hsi ==> his
cpu/stm32/dist/clk_conf/clk_conf.h:571: hsi ==> his
cpu/stm32/dist/clk_conf/clk_conf.h:586: hsi ==> his
cpu/stm32/doc.txt:16: HSI ==> HIS
cpu/stm32/doc.txt:31: HSI ==> HIS
cpu/stm32/doc.txt:32: HSI ==> HIS
cpu/stm32/doc.txt:33: HSI ==> HIS
cpu/stm32/include/periph_cpu.h:367: ot ==> to, of, or
cpu/stm32/include/periph_cpu.h:367: ot ==> to, of, or
cpu/stm32/include/stmclk.h:34: HSI ==> HIS
cpu/stm32/include/stmclk.h:35: HSI ==> HIS
cpu/stm32/include/stmclk.h:40: HSI ==> HIS
cpu/stm32/include/stmclk.h:47: HSI ==> HIS
cpu/stm32/include/stmclk.h:52: HSI ==> HIS
cpu/stm32/include/stmclk.h:54: HSI ==> HIS
cpu/stm32/periph/adc_l1.c:61: HSI ==> HIS
cpu/stm32/periph/adc_l1.c:64: HSI ==> HIS
cpu/stm32/periph/flashpage.c:91: HSI ==> HIS
cpu/stm32/periph/flashpage.c:146: HSI ==> HIS
cpu/stm32/periph/flashpage.c:182: HSI ==> HIS
cpu/stm32/periph/flashpage.c:221: HSI ==> HIS
cpu/stm32/periph/uart.c:320: HSI ==> HIS
cpu/stm32/periph/uart.c:433: UE ==> USE, DUE
cpu/stm32/periph/uart.c:434: UE ==> USE, DUE
cpu/stm32/periph/uart.c:445: UE ==> USE, DUE
cpu/stm32/stmclk/stmclk_common.c:67: HSI ==> HIS
cpu/stm32/stmclk/stmclk_f0f1f3.c:33: HSI ==> HIS
cpu/stm32/stmclk/stmclk_f0f1f3.c:120: HSI ==> HIS
cpu/stm32/stmclk/stmclk_f0f1f3.c:121: HSI ==> HIS
cpu/stm32/stmclk/stmclk_f0f1f3.c:122: HSI ==> HIS
cpu/stm32/stmclk/stmclk_f0f1f3.c:146: HSI ==> HIS
cpu/stm32/stmclk/stmclk_f0f1f3.c:149: HSI ==> HIS
cpu/stm32/stmclk/stmclk_f0f1f3.c:161: HSI ==> HIS
cpu/stm32/stmclk/stmclk_f0f1f3.c:198: HSI ==> HIS
cpu/stm32/stmclk/stmclk_f2f4f7.c:135: HSI ==> HIS
cpu/stm32/stmclk/stmclk_f2f4f7.c:138: HSI ==> HIS
cpu/stm32/stmclk/stmclk_f2f4f7.c:147: HSI ==> HIS
cpu/stm32/stmclk/stmclk_gx.c:167: HSI ==> HIS
cpu/stm32/stmclk/stmclk_gx.c:172: HSI ==> HIS
cpu/stm32/stmclk/stmclk_gx.c:191: HSI ==> HIS
cpu/stm32/stmclk/stmclk_l0l1.c:67: HSI ==> HIS
cpu/stm32/stmclk/stmclk_l0l1.c:143: HSI ==> HIS
cpu/stm32/stmclk/stmclk_l0l1.c:144: HSI ==> HIS
cpu/stm32/stmclk/stmclk_l0l1.c:145: HSI ==> HIS
cpu/stm32/stmclk/stmclk_l0l1.c:178: HSI ==> HIS
cpu/stm32/stmclk/stmclk_l0l1.c:185: HSI ==> HIS
cpu/stm32/stmclk/stmclk_l0l1.c:226: HSI ==> HIS
cpu/stm32/stmclk/stmclk_l0l1.c:249: HSI ==> HIS
cpu/stm32/stmclk/stmclk_l4wb.c:194: HSI ==> HIS
cpu/stm32/stmclk/stmclk_l4wb.c:197: HSI ==> HIS
cpu/stm32/stmclk/stmclk_l4wb.c:212: HSI ==> HIS
cpu/stm32/stmclk/stmclk_l4wb.c:291: HSI ==> HIS
dist/tools/buildsystem_sanity_check/check.sh:76: files' ==> file's
dist/tools/esptool/espreset.py:62: ser ==> set
dist/tools/esptool/espreset.py:63: ser ==> set
dist/tools/esptool/espreset.py:65: ser ==> set
dist/tools/esptool/espreset.py:66: ser ==> set
dist/tools/esptool/espreset.py:70: ser ==> set
dist/tools/esptool/espreset.py:71: ser ==> set
dist/tools/esptool/esptool.py:2604: files' ==> file's
dist/tools/mcuboot/imgtool.md:20: keypair ==> key pair
dist/tools/suit/suit-manifest-generator/README.md:183: fo ==> of, for
dist/tools/suit/suit-manifest-generator/README.md:215: keypair ==> key pair
dist/tools/suit/suit-manifest-generator/suit_tool/argparser.py:57: formated ==> formatted
dist/tools/suit/suit-manifest-generator/suit_tool/compile.py:376: te ==> the, be
doc/doxygen/src/advanced-build-system-tricks.md:180: adminstrative ==> administrative
doc/doxygen/src/kconfig/kconfig.md:103: Ouput ==> Output
drivers/at86rf215/at86rf215.c:207: AACK ==> ACK
drivers/at86rf215/at86rf215_netdev.c:706: AACK ==> ACK
drivers/at86rf215/at86rf215_netdev.c:995: tHW ==> the, thaw
drivers/at86rf2xx/at86rf2xx_netdev.c:835: neccessary ==> necessary
drivers/enc28j60/enc28j60.c:86: Preamle ==> Preamble
drivers/include/at.h:35: correspondant ==> correspondent
drivers/include/ina3221.h:56: measurment ==> measurement
drivers/include/ina3221.h:372: enble ==> enable
drivers/include/lc709203f.h:314: Invlid ==> Invalid
drivers/include/lpsxxx.h:139: pres ==> press
drivers/include/lpsxxx.h:144: pres ==> press
drivers/include/ltc4150.h:156: tread ==> thread, treat
drivers/include/mhz19.h:34: Manal ==> Manual
drivers/include/mhz19.h:34: manal ==> manual
drivers/include/mpl3115a2.h:164: pres ==> press
drivers/include/mpl3115a2.h:170: pres ==> press
drivers/include/periph/gpio.h:200: occured ==> occurred
drivers/include/pir.h:126: whan ==> want, when
drivers/include/stmpe811.h:95: inout ==> input, in out
drivers/lpsxxx/lpsxxx.c:157: pres ==> press
drivers/lpsxxx/lpsxxx.c:195: pres ==> press
drivers/mhz19/include/mhz19_params.h:18: Manal ==> Manual
drivers/mhz19/include/mhz19_params.h:18: manal ==> manual
drivers/mhz19/mhz19_pwm.c:18: Manal ==> Manual
drivers/mhz19/mhz19_pwm.c:18: manal ==> manual
drivers/mhz19/mhz19_uart.c:19: Manal ==> Manual
drivers/mhz19/mhz19_uart.c:19: manal ==> manual
drivers/mpl3115a2/mpl3115a2.c:155: pres ==> press
drivers/mpl3115a2/mpl3115a2.c:168: pres ==> press
drivers/mpl3115a2/mpl3115a2.c:169: pres ==> press
drivers/mpl3115a2/mpl3115a2.c:169: pres ==> press
drivers/mtd_mci/mtd_mci.c:2: Technik ==> Technique
drivers/mtd_mci/mtd_mci.c:34: suports ==> supports
drivers/sdp3x/sdp3x_saul.c:40: pres ==> press
drivers/sdp3x/sdp3x_saul.c:44: pres ==> press
drivers/w5100/w5100.c:105: extint ==> extinct, extant
drivers/w5100/w5100.c:124: extint ==> extinct, extant
examples/openthread/main.c:18: ot ==> to, of, or
examples/wakaama/main.c:17: Manal ==> Manual
examples/wakaama/main.c:17: manal ==> manual
pkg/nimble/netif/include/nimble_netif.h:124: mater ==> matter, master, mother
pkg/nordic_softdevice_ble/README-BLE-6LoWPAN.md:1: connecton ==> connection, connector
pkg/openthread/contrib/netdev/openthread_netdev.c:31: ot ==> to, of, or
pkg/openthread/contrib/openthread.c:23: ot ==> to, of, or
pkg/openthread/contrib/platform_alarm.c:23: ot ==> to, of, or
pkg/openthread/contrib/platform_functions_wrapper.c:13: OT ==> TO, OF, OR
pkg/openthread/contrib/platform_functions_wrapper.c:13: OT ==> TO, OF, OR
pkg/openthread/contrib/platform_functions_wrapper.c:27: ot ==> to, of, or
pkg/openthread/contrib/platform_functions_wrapper.c:222: OT ==> TO, OF, OR
pkg/openthread/contrib/platform_misc.c:22: ot ==> to, of, or
pkg/openthread/contrib/platform_radio.c:32: ot ==> to, of, or
pkg/openthread/contrib/platform_uart.c:27: ot ==> to, of, or
pkg/openthread/include/ot.h:170: OT ==> TO, OF, OR
pkg/wakaama/contrib/lwm2m_client.c:17: Manal ==> Manual
pkg/wakaama/contrib/lwm2m_client.c:17: manal ==> manual
pkg/wakaama/contrib/lwm2m_client_connection.c:16: Manal ==> Manual
pkg/wakaama/contrib/lwm2m_client_connection.c:36: Manal ==> Manual
pkg/wakaama/contrib/lwm2m_client_connection.c:36: manal ==> manual
pkg/wakaama/contrib/lwm2m_platform.c:15: Manal ==> Manual
pkg/wakaama/contrib/lwm2m_platform.c:26: Manal ==> Manual
pkg/wakaama/contrib/lwm2m_platform.c:26: manal ==> manual
pkg/wakaama/contrib/objects/device.c:15: Manal ==> Manual
pkg/wakaama/contrib/objects/device.c:15: manal ==> manual
pkg/wakaama/include/lwm2m_client_config.h:22: Manal ==> Manual
pkg/wakaama/include/lwm2m_client_config.h:22: manal ==> manual
pkg/wakaama/include/lwm2m_client_config.h:186: bindins ==> bindings
pkg/wakaama/include/lwm2m_client_connection.h:16: Manal ==> Manual
pkg/wakaama/include/lwm2m_client_connection.h:36: Manal ==> Manual
pkg/wakaama/include/lwm2m_client_connection.h:36: manal ==> manual
release-notes.txt:1985: seperator ==> separator
release-notes.txt:1986: seperator ==> separator
release-notes.txt:2860: build-in ==> built-in
release-notes.txt:3085: crashs ==> crashes
release-notes.txt:4312: witth ==> with
release-notes.txt:4404: unconsistent ==> inconsistent
release-notes.txt:4605: witth ==> with
release-notes.txt:4654: unconsistent ==> inconsistent
release-notes.txt:5204: hundret ==> hundred, hundreds
release-notes.txt:5204: agressive ==> aggressive
release-notes.txt:5217: GUAs ==> gauss
release-notes.txt:5261: Eratic ==> Erratic
release-notes.txt:5692: temperatur ==> temperature
release-notes.txt:5744: hundret ==> hundred, hundreds
release-notes.txt:5744: agressive ==> aggressive
release-notes.txt:5755: GUAs ==> gauss
release-notes.txt:5811: Eratic ==> Erratic
release-notes.txt:6044: hundret ==> hundred, hundreds
release-notes.txt:6044: agressive ==> aggressive
release-notes.txt:6054: GUAs ==> gauss
release-notes.txt:6094: numberic ==> numeric
release-notes.txt:6116: Eratic ==> Erratic
release-notes.txt:6309: GUAs ==> gauss
release-notes.txt:6492: Sytem ==> System
release-notes.txt:6534: GUAs ==> gauss
release-notes.txt:7340: registeres ==> registers
sys/arduino/include/spiport.hpp:169: inout ==> input, in out
sys/arduino/include/wireport.hpp:78: adress ==> address
sys/can/device.c:506: theoritical ==> theoretical
sys/cpp11-compat/include/riot/mutex.hpp:305: inout ==> input, in out
sys/cpp11-compat/include/riot/mutex.hpp:306: inout ==> input, in out
sys/cpp11-compat/include/riot/thread.hpp:271: inout ==> input, in out
sys/cpp11-compat/include/riot/thread.hpp:319: inout ==> input, in out
sys/cpp11-compat/include/riot/thread.hpp:320: inout ==> input, in out
sys/include/can/conn/isotp.h:94: inout ==> input, in out
sys/include/can/conn/isotp.h:127: inout ==> input, in out
sys/include/can/conn/isotp.h:139: inout ==> input, in out
sys/include/can/conn/raw.h:69: inout ==> input, in out
sys/include/fmt.h:514: inout ==> input, in out
sys/include/matstat.h:107: inout ==> input, in out
sys/include/net/gnrc/netif.h:53: dedup ==> dedupe
sys/include/net/gnrc/netif.h:79: Busses ==> Buses
sys/include/net/gnrc/netif/lorawan.h:48: wether ==> weather, whether
sys/include/net/gnrc/rpl/srh.h:47: comprE ==> compare, compère
sys/include/net/gnrc/tcp.h:61: inferface ==> interface
sys/include/net/ipv4/hdr.h:169: fo ==> of, for
sys/include/net/ipv4/hdr.h:171: fo ==> of, for
sys/include/net/ipv4/hdr.h:174: fo ==> of, for
sys/include/net/ipv4/hdr.h:175: fo ==> of, for
sys/include/xtimer.h:413: occcured ==> occurred
sys/include/xtimer.h:424: occcured ==> occurred
sys/include/zptr.h:15: bounderies ==> boundaries
sys/include/zptr.h:86: compressable ==> compressible
sys/include/zptr.h:101: Substracts ==> Subtracts
sys/include/ztimer/periodic.h:101: inout ==> input, in out
sys/net/gnrc/network_layer/ipv6/nib/_nib-arsm.c:237: registartion ==> registration
sys/net/gnrc/network_layer/sixlowpan/frag/gnrc_sixlowpan_frag.c:151: divisable ==> divisible
sys/net/gnrc/transport_layer/tcp/gnrc_tcp_fsm.c:309: anounce ==> announce
sys/posix/include/fcntl.h:59: creat ==> create
sys/posix/pthread/pthread_tls.c:185: defore ==> before
sys/random/fortuna/fortuna.c:261: inout ==> input, in out
sys/random/fortuna/fortuna.c:268: inout ==> input, in out
sys/random/fortuna/fortuna.c:275: inout ==> input, in out
sys/random/fortuna/fortuna.h:160: inout ==> input, in out
sys/random/fortuna/fortuna.h:171: inout ==> input, in out
sys/random/fortuna/fortuna.h:186: inout ==> input, in out
sys/random/fortuna/fortuna.h:206: inout ==> input, in out
sys/random/fortuna/fortuna.h:220: inout ==> input, in out
sys/random/fortuna/fortuna.h:221: inout ==> input, in out
sys/random/fortuna/fortuna.h:225: inout ==> input, in out
sys/shell/commands/sc_cryptoauthlib.c:131: UE ==> USE, DUE
sys/shell/commands/sc_sht1x.c:192: presicion ==> precision
sys/ztimer/core.c:16: functinality ==> functionality
tests/bench_timers/print_results.c:111: pres ==> press
tests/bench_timers/print_results.h:57: pres ==> press
tests/bench_xtimer/README.md:17: emtpy ==> empty
tests/driver_ina3221/main.c:30: thist ==> this
tests/driver_lpsxxx/main.c:40: pres ==> press
tests/driver_lpsxxx/main.c:47: pres ==> press
tests/driver_lpsxxx/main.c:54: pres ==> press
tests/driver_mhz19/main.c:19: Manal ==> Manual
tests/driver_mhz19/main.c:19: manal ==> manual
tests/gnrc_dhcpv6_client_6lbr/README.md:25: sting ==> string
tests/gnrc_ipv6_ext/README.md:27: sting ==> string
tests/gnrc_ipv6_ext_frag/README.md:27: sting ==> string
tests/gnrc_ipv6_ext_opt/README.md:28: sting ==> string
tests/gnrc_rpl_srh/README.md:25: sting ==> string
tests/ieee802154_hal/main.c:637: AACK ==> ACK
tests/od/main.c:32: discrepency ==> discrepancy
tests/periph_i2c/main.c:45: consumtion ==> consumption
tests/periph_timer_periodic/main.c:2: Technik ==> Technique
tests/pkg_c25519/main.c:44: keypair ==> key pair
tests/pkg_hacl/main.c:52: keypair ==> key pair
tests/pkg_hacl/main.c:55: keypair ==> key pair
tests/pkg_monocypher/main.c:42: keypair ==> key pair
tests/pkg_openwsn/README.md:211: BA ==> BY, BE
tests/pkg_openwsn/README.md:254: BA ==> BY, BE
tests/pkg_qdsa/main.c:51: keypair ==> key pair
tests/pkg_tweetnacl/main.c:63: keypair ==> key pair
tests/pkg_tweetnacl/main.c:66: keypair ==> key pair
tests/pkg_tweetnacl/main.c:97: keypair ==> key pair
tests/pthread_cooperation/main.c:28: ths ==> the, this
tests/pthread_cooperation/main.c:61: ths ==> the, this
tests/pthread_cooperation/main.c:66: ths ==> the, this
tests/shell/tests/01-run.py:79: te ==> the, be
tests/shell/tests/01-run.py:83: hel ==> help, hell, heal
tests/sys_crypto/tests-crypto-modes-ccm.c:895: neccessary ==> necessary
tests/thread_cooperation/main.c:64: ths ==> the, this
tests/thread_cooperation/main.c:73: ths ==> the, this
tests/thread_cooperation/main.c:78: ths ==> the, this
tests/thread_cooperation/main.c:83: ths ==> the, this
tests/thread_cooperation/main.c:85: ths ==> the, this
tests/unittests/tests-hashes/tests-hashes-sha1.c:54: BA ==> BY, BE
tests/unittests/tests-hashes/tests-hashes-sha1.c:55: BA ==> BY, BE
tests/unittests/tests-hashes/tests-hashes-sha1.c:107: BA ==> BY, BE
tests/unittests/tests-sock_util/tests-sock_util.c:52: doesnt ==> doesn't, does not
tests/unittests/tests-sock_util/tests-sock_util.c:55: doesnt ==> doesn't, does not
```

</details>

### Issues/PRs references

None

### CODE REVIEW

File: dist/tools/suit/suit-manifest-generator/suit_tool/compile.py [hunk 1]
```diff
@@ -373,7 +373,6 @@ def compile_manifest(options, m):
                 'command-arg' : None
             }))
         else:
-            te = []
```
Comment (fjmolinas) [inline dist/tools/suit/suit-manifest-generator/suit_tool/compile.py:376]: Can you remove this? This is external code.

File: release-notes.txt [hunk 2]
```diff
@@ -1982,8 +1982,8 @@ API Changes
 - static inline int ipv6_addr_split_iface(char *addr_str)
 + static inline char *ipv6_addr_split_iface(char *addr_str)
 
-- int ipv6_addr_split(char *addr_str, char seperator, int _default);
-+ int ipv6_addr_split_int(char *addr_str, char seperator, int _default);
+- int ipv6_addr_split(char *addr_str, char separator, int _default);
```
Comment (fjmolinas) [inline release-notes.txt:1985]: Should we do this for release notes?

Comment (aabadie) [reply release-notes.txt:1985]: TBH, I wasn't sure and even thought of excluding this file from codespell check. But I think it's important to also process this file.
For this particular change, I checked the API and the parameter is `separator` in RIOT. So the typo was fixed in the code already.

File: dist/tools/suit/suit-manifest-generator/suit_tool/compile.py (hunk 1 above)
Comment (aabadie) [reply dist/tools/suit/suit-manifest-generator/suit_tool/compile.py:376]: My editor is reporting that the variable is not used (which is true) that's why I removed it.

Anyway, this change will be useless after we merge #15095.

File: release-notes.txt (hunk 2 above)
Comment (fjmolinas) [reply release-notes.txt:1985]: Ok!

File: dist/tools/suit/suit-manifest-generator/suit_tool/compile.py (hunk 1 above)
Comment (fjmolinas) [reply dist/tools/suit/suit-manifest-generator/suit_tool/compile.py:376]: Ok, no harm with it I guess.

File: release-notes.txt (hunk 2 above)
Comment (fjmolinas) [reply release-notes.txt:1985]: > But I think it's important to also process this file.

I think its important, I'm just unsure if we should do it for passed release notes.

File: dist/tools/suit/suit-manifest-generator/suit_tool/compile.py (hunk 1 above)
Comment (aabadie) [reply dist/tools/suit/suit-manifest-generator/suit_tool/compile.py:376]: You can run the test for `examples/suit_update` to be sure.

File: release-notes.txt (hunk 2 above)
Comment (aabadie) [reply release-notes.txt:1985]: > I'm just unsure if we should do it for passed release notes

I think the same but it's technically difficult, codespell can only check a whole file or ignore it.

Comment (fjmolinas) [reply release-notes.txt:1985]: Lets get some other input on this, I personally don't mind doing it though.

**Review by fjmolinas — APPROVED** [review]
ACK!

**Comment by aabadie** [comment]
Thanks @fjmolinas !

**Pull request merged**